server/src/routes/follow.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const Follows = require('../db/mongo/models/follow');
+
+const router = express.Router();
+
+router.get('/:email_from/followers', async (req, res) => {
```

Comment:
Add validation in the beginning of every route, something similar to `validateFollowInput` in my PR:
`email_from` valid (exists and not empty), `email_to` valid (exists and not empty) depends on the route.